Makefile

```diff
@@ -202,3 +202,6 @@ JSONNETFMT_CMD := $(JSONNETFMT) -n 2 --max-blank-lines 2 --string-style s --comm
 .PHONY: jsonnet-fmt
 jsonnet-fmt: | $(JSONNETFMT)
 	PATH=$$PATH:$(BIN_DIR):$(FIRST_GOPATH)/bin echo ${JSONNET_SRC} | xargs -n 1 -- $(JSONNETFMT_CMD) -i
+
+generate-rules-code:
```

Comment:
Yes, @jessicalins the missing piece here is that you need to add `$(OAPI_CODEGEN)` as a dependency of the generation target, otherwise the binary will not be guaranteed to exist

Reply:
thanks! added now